package.json

```
@@ -1,63 +1,70 @@
 {
-  "name": "14_sprint",
+  "name": "15_sprint",
   "version": "1.0.1",
-  "description": "14_sprint",
+  "description": "15_sprint",
   "main": "app.js",
   "scripts": {
+    "test": "jest",
     "start": "node app.js",
```

Нужно исправить: в продакшене и в режиме разработки используется один и тот же секректный ключ.
Если `.env` файл на сервере создан и заполнен, то скорее всего проблема в том, что он не подключается. Чтобы его подключить сразу рекомендую подключать его прямо в команде запуска:
```suggestion
    "start": "node -r dotenv/config app.js",
```
Флаг `-r` используется для того, чтобы подключить модуль до запуска основного модуля:
```bash
$ node --help
  -r, --require=...                         module to preload (option can be repeated)
```